src/pages/ProfilePage/ProfileComponent/ListComponent.js

```diff
@@ -14,6 +14,10 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 export default function ListComponent() {
+  // const [checked, setChecked] = React.useState(false);
+  // const toggleChecked = () => {
+  //   setChecked((prev) => !prev);
+  // };
   const classes = useStyles();
```

Comment:
Please Revert it.

Reply:
But we are not using these anywhere thats why it is showing error